Source/AwsCommonRuntimeKit/crt/CBOR.swift

```diff
@@ -63,34 +63,47 @@ public class CBOREncoder {
 
     /// Encode a single type
     /// - Parameters:
-    ///   - value: value to encode 
+    /// - value: value to encode
     /// - Throws: CommonRuntimeError.crtError
     public func encode(_ value: CBORType) {
         switch value {
-        case .uint(let value): aws_cbor_encoder_write_uint(self.rawValue, value)
-        case .int(let value):
-            do {
-                if value >= 0 {
-                    aws_cbor_encoder_write_uint(self.rawValue, UInt64(value))
-                } else {
-                    aws_cbor_encoder_write_negint(self.rawValue, UInt64(-1 - value))
-                }
+        case .uint(let v):
+            aws_cbor_encoder_write_uint(self.rawValue, v)
```

Comment:
trivial: Personally, I don't like single-character variable names because they do not convey much meaning. For example, `d` could represent data or a date. Can we rename this to a more descriptive name instead?